entrypoint

```diff
@@ -1,32 +1,8 @@
 #!/usr/bin/env sh
 
-echo " "
-echo "==================================================================================================="
-echo "==================================================================================================="
-
-source /opt/conda/etc/profile.d/conda.sh
-
 git config --global user.name "conda-forge-webservices[bot]"
 git config --global user.email "91080706+conda-forge-webservices[bot]@users.noreply.github.com"
 
-conda activate base
-conda config --set show_channel_urls True
-conda config --add channels conda-forge
-conda config --remove channels defaults || true
-conda config --set channel_priority strict
-conda config --set always_yes yes
-
-conda config --show-sources
-
-conda update --all --yes
```

Comment:
We should not remove this until we understand why it was there. 

One issue is that smithy checks if it is update to date before rerendering IIRC? We'd need to turn that off or there will be a race condition when a smithy version is pushed.

Reply:
But how can we even be sure that the PyPI installed packages still work in this case? Since conda doesn't check against their dependencies AFAIK. I think this is _very_ bad practice.

Reply:
I am not going to disagree with that! :)

We run the live tests before anything merges and those test that the version update command from the bot package still works. This prevents that error.

I have an open issue to fix this up: https://github.com/regro/cf-scripts/issues/2894

Reply:
I run the conda install against the bot deps here BTW:

```
micromamba install --yes --quiet --name base --file cf-scripts/environment.yml && \
micromamba install --yes --quiet --name base --file /tmp/environment.yml
```

That appears to be missing.